src/javaExercises/SortMatrix/SortMatrix.java

```diff
@@ -0,0 +1,85 @@
+package javaExercises.SortMatrix;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class SortMatrix {
+    public int[][] populateArrayWithRandomValues(int n, int max, int min) {
```

Comment:
1. give a more meaningful name to your variables
2. i think that min and max variables are redundant here, just set some basic values

Reply:
Done